editor/components/rich-text/tinymce.js

```diff
@@ -202,8 +202,12 @@ export default class TinyMCE extends Component {
 	render() {
 		const { tagName = 'div', style, defaultValue, className, isPlaceholderVisible, format } = this.props;
 		const ariaProps = pickAriaProps( this.props );
-		if ( [ 'ul', 'ol', 'table' ].indexOf( tagName ) === -1 ) {
+
+		if ( tagName !== 'table' ) {
 			ariaProps.role = 'textbox';
+		} else {
+			// The `aria-multiline` attribute must be used only together with `role=textbox`.
+			ariaProps[ 'aria-multiline' ] = null;
```

Comment:
I think we discussed that we should almost always use `aria-multiline`. So what if we remove it entirely from the `ariaProps` and just add it here where we add `ariaProps.role = 'textbox';`. This seems consistent for me?

Reply:
Yep, we missed the table case... 😬 Still, we should allow to use both `true` (emulating a textarea) and `false` (emulating an input field). Omit it entirely when there's no role=textbox.

Reply:
Agreed on this one, we should be setting properties rather than removing it 👍

Reply:
Sure, agreed to simplify and always handle aria-multiline internally setting it to `true` as TinyMCE always behaves like a textarea, where text naturally wraps in multiple lines (regardless of the Enter key behavior). Removed the prop from the readme doc.